packages/manager/modules/pci/src/projects/project/private-registry/private-registry.html

```diff
@@ -87,30 +88,31 @@
                     data-external
                     data-href="{{:: $row.url }}"
                     data-disabled="$row.status !== $ctrl.PRIVATE_REGISTRY_STATUS.READY"
+                    data-on-click="$ctrl.trackClick('PCI_PROJECTS_PRIVATEREGISTRY_HARBOR-UI')"
                 >
                     <span data-translate="private_registry_harbor_ui"></span>
                 </oui-action-menu-item>
                 <oui-action-menu-item
-                    data-on-click="$ctrl.copyApiUrl($row.id, $row.url)"
+                    data-on-click="$ctrl.copyApiUrl($row.id, $row.url); $ctrl.trackClick('PCI_PROJECTS_PRIVATEREGISTRY_API-URL')"
                     data-disabled="$row.status !== $ctrl.PRIVATE_REGISTRY_STATUS.READY"
                 >
                     <span data-translate="private_registry_harbor_api"></span>
                 </oui-action-menu-item>
                 <oui-action-menu-item
-                    data-on-click="$ctrl.generateCredentials($row.id, $row.name, $row.url)"
+                    data-on-click="$ctrl.generateCredentials($row.id, $row.name, $row.url); $ctrl.trackClick('PCI_PROJECTS_PRIVATEREGISTRY_CREDENTIALS')"
                     data-disabled="$row.status !== $ctrl.PRIVATE_REGISTRY_STATUS.READY"
                 >
                     <span
                         data-translate="private_registry_regenerate_creds"
                     ></span>
                 </oui-action-menu-item>
                 <oui-action-menu-item
-                    data-on-click="$ctrl.updateRegistry($row.id, $row.name)"
+                    data-on-click="$ctrl.updateRegistry($row.id, $row.name); $ctrl.trackClick('PCI_PROJECTS_PRIVATEREGISTRY_UPDATE')"
                 >
                     <span data-translate="private_registry_rename"></span>
                 </oui-action-menu-item>
                 <oui-action-menu-item
-                    data-on-click="$ctrl.deleteRegistry($row.id, $row.name)"
+                    data-on-click="$ctrl.deleteRegistry($row.id, $row.name); $ctrl.trackClick('PCI_PROJECTS_PRIVATEREGISTRY_DELETE')"
```

Comment:
do you want to track on delete initiate or on delete page visit? I think this takes to delete dialog page right?

Reply:
Aside of @radireddy remark, @varun257 if you choose to set the tracking on visit page, you will have to change the type of track, from `atInternet.trackClick` call to `atInternet.trackPage` one

Reply:
Already have a `trackPage` for this. This is in addition to the page tracking. This is specifically to track click event of the button.